### ISSUE TITLE
Bug in newLinesRule

### ISSUE THREAD

**Issue by Cheshiriks** [diktat-static-analysis/diKTat#1065]
The bug is triggered when starting the diktat together with the ktlint
Bug in `Example2Test.kt` in smoke tests in diktat 1.0.0-rc.3 and older
```
java.lang.NullPointerException: Cannot invoke "org.jetbrains.kotlin.com.intellij.lang.ASTNode.getElementType()" because the return value of "org.jetbrains.kotlin.com.intellij.lang.ASTNode.getFirstChildNode()" is null
        at org.cqfn.diktat.ruleset.rules.chapter3.files.NewlinesRule.handleOperatorWithLineBreakAfter(NewlinesRule.kt:152)
        at org.cqfn.diktat.ruleset.rules.chapter3.files.NewlinesRule.logic(NewlinesRule.kt:125)
        at org.cqfn.diktat.ruleset.rules.DiktatRule.visit(DiktatRule.kt:50)
        at com.pinterest.ktlint.core.KtLint$format$1.invoke(KtLint.kt:271)
        at com.pinterest.ktlint.core.KtLint$format$1.invoke(KtLint.kt:23)
        at com.pinterest.ktlint.core.KtLint$visitor$2$2.invoke(KtLint.kt:212)
        at com.pinterest.ktlint.core.KtLint$visitor$2$2.invoke(KtLint.kt:23)
```

**Comment by akuleshov7** [comment]
@sanyavertolet please test it with a new version of diktat and new ktlint (via console)

**Comment by sanyavertolet** [comment]
Cannot reproduce bug with diktat demo.

**Comment by sanyavertolet** [comment]
Reproduced it with the newest version of diktat and ktlint. The bug is still there.

**Comment by akuleshov7** [comment]
Please check it and find the reason @sanyavertolet

**Comment by petertrr** [comment]
@sanyavertolet is it still reproducible with `--disabled_rules=standard`?

**Comment by sanyavertolet** [comment]
@petertrr nope

**Comment by petertrr** [comment]
> @petertrr nope

@akuleshov7 should we consider it resolved then? IIRC, we have been waiting for ktlint update for this very reason, to separate ourselves from these interop issues

**Comment by akuleshov7** [comment]
Guys, come on :))
Of course than it is not an issue
